Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "either"
 version = "1.13.0"
 authors = ["bluss"]
 edition = "2018"
```

Comment:
We could also upgrade to 2021, if you're so inclined.

Reply:
Sure, done.
I thought about that but wasn't sure you'd want that since `itertools` is still on 2018.

Reply:
Maybe I'll just upgrade `itertools` to 2021 next 😅

Reply:
Thankfully, this decision can be made independently! :)
(I'm not a member of `itertools`, but I expect they'll be fine with that.)